math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -273,6 +273,17 @@ public:
       return ret;
    }
 
+   RVec<T> operator[](const RVec<size_type> &idx) const
```

Comment:
Do we need to sanitize the template specialization with `enable_if` or other template magic? is it too confusing to have `v[idx]` and `v[v>0]`?